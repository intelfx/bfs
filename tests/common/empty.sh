bfs_diff basic -empty
