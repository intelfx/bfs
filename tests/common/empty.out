basic/a
basic/b
basic/c/d
basic/e/f
basic/g/h
basic/i
basic/j/foo
basic/k/foo/bar
